ksqldb-engine/src/main/java/io/confluent/ksql/exception/KsqlGroupAuthorizationException.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.exception;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.kafka.common.acl.AclOperation;
+import org.apache.kafka.common.errors.TopicAuthorizationException;
+
+/**
+ * Used to return custom error messages when TopicAuthorizationException returned from Kafka
+ */
+public class KsqlGroupAuthorizationException extends TopicAuthorizationException {
```

Comment:
This inheritance smells!  A group is not a topic. So why inherit from `TopicAuthorizationException`?  Maybe consider inheriting from `GroupAuthorizationException`?

Reply:
ack